src/types/responses/AccountBalanceInfo.ts

```diff
@@ -28,6 +28,15 @@ export interface IAccountBalanceInfo {
 	miscFrozen: Balance | string;
 	feeFrozen: Balance | string;
 	frozen: Balance | string;
+	/**
+	 * Calculated transferable balance. This uses the formula: free - max(maybeEd, frozen - reserve)
+	 * Where `maybeEd` means if there is no frozen and reserves it will be zero, else it will be the existential deposit.
```

Comment:
This description actually needs to be adjusted. If no frozen or reserve it will be MAX(ED, frozen - reserve)